lib/cassia/response_handlers/connect_device.rb

```diff
@@ -0,0 +1,28 @@
+module Cassia
+  module ResponseHandlers
+    class ConnectDevice
+      def initialize(access_controller)
+        @access_controller = access_controller
+      end
+
+      def handle(response)
+        if response.success?
+          handle_success(response)
+        else
+          handle_failure(response)
+        end
+        response.success?
+      end
+
+      private
+
+      def handle_success(response)
+        @access_controller.device_mac = response.body['device']
```

Comment:
this will change based on my advice above to use the `Device` model to hold `connected_devices` and append to a list